node-process-argv/echo.js

```diff
@@ -0,0 +1 @@
+console.log(process.argv.slice(2).join(' '));
```

Comment:
Clever way to get all the arguments, not just the first!

Reply:
I misunderstood the prompt and thought all the arguments that were listed in the terminal were to be console logged haha